src/main/java/net/atos/zac/mail/MailService.java

```diff
@@ -237,8 +247,8 @@ private byte[] createPdfDocument(final String verzender, final String ontvanger,
 
     private void addToParagraph(final Paragraph paragraph, final String propertie, final String inhoud) {
         paragraph.add(String.format("%s: %s", propertie, inhoud));
```

Comment:
I believe we can use `%n` as new line character when using `String.format`, that would eliminate the next 2 lines.